src/commands.cpp

```diff
@@ -2867,6 +2867,28 @@ bool CommandPutUAVer::procresult(Result r)
                     LOG_err << "Failed to decrypt " << User::attr2string(at) << " after putua";
                 }
             }
+            else if (at == ATTR_BACKUP_NAMES && client->mSendingBackupName)
+            {
+                if (client->mPendingBackupNames.empty())
+                {
+                    client->mSendingBackupName = false;
+                }
+                else    // more names arrived during `upv`
+                {
+                    string_map attrMap = move(client->mPendingBackupNames);
```

Comment:
why not use the mPendingBackupNames directly and have it emptied after User::mergeUserAttribute?

Reply:
Agree. Fixed.